SixTrack/sixtrack.s

```diff
@@ -1419,6 +1419,16 @@ C     Block with data/fields needed for checkpoint/restart of DYNK
 !
 !-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-
 !
++cd zipf
+      integer zipf_maxfiles,zipf_numfiles
+      parameter (zipf_maxfiles=64)
```

Comment:
Surely when say debugging and you want *everything* back 64 might be a little low?